apps/frontend/src/api/__tests__/util.test.tsx

```diff
@@ -100,7 +100,9 @@ describe('useFetchOptions', () => {
 
 describe('createListApiUrl', () => {
   it('handles requests without parameters ', async () => {
-    expect(createListApiUrl('test').toString()).toMatch(/\/test$/);
+    expect(createListApiUrl('test').toString()).toMatch(
```

Comment:
come on, don't just go for what requires the least lines changed :see_no_evil: :D
it handles requests without parameters (checks the pathname ends with test as before)
it appends the default pagination params
it selects the given page